### PR TITLE
Add healthcheck php-fpm

### LINKED ISSUE #1295: Include healthcheck for php container out of the box
**Description**  
Currently, we don't include a healthcheck for the `php` container out of the box. This could be a pain when the container doesn't start for any reason, and the container orchestrator doesn't know when to kill it (in the entrypoint script, we wait for the database to become available), so the service would remain unavailable.

See discussion from https://github.com/api-platform/api-platform/pull/1289#issuecomment-542633436 onwards.

**Example**  
N/A

### PR DESCRIPTION
**Pull request by patrick-mota**
| Q             | A
| ------------- | ---
| Bug fix?      | no 
| New feature?  | yes 
| BC breaks?    | no
| Deprecations? | no
| Tickets       | fixes #1295  
| License       | MIT
| Doc PR        | 

Add healthcheck in php container see #1295

### CODE REVIEW

**Comment by dunglas** [comment]
Could you also add a similar `livenessProbe` and a `readynessProbe` in (https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/) in the Helm chart? https://github.com/api-platform/api-platform/blob/master/api/helm/api/templates/php-deployment.yaml (/cc @pborreli)

File: api/Dockerfile [hunk 1]
```diff
@@ -18,6 +18,7 @@ RUN apk add --no-cache \
 		file \
 		gettext \
 		git \
+		fcgi \
```
Comment (teohhanhui) [inline api/Dockerfile]: Please sort alphabetically.

File: api/Dockerfile [hunk 2]
```diff
@@ -54,6 +55,13 @@ RUN set -eux; \
 	apk add --no-cache --virtual .api-phpexts-rundeps $runDeps; \
 	\
 	apk del .build-deps
+	
+HEALTHCHECK --interval=10s --timeout=3s \
+   	CMD \
```
Comment (teohhanhui) [inline api/Dockerfile]: Please be mindful of the whitespace in this file. Use tabs, not spaces, for indentation.

File: api/Dockerfile [hunk 3]
```diff
@@ -54,6 +55,13 @@ RUN set -eux; \
...
+    SCRIPT_NAME=/ping \
```
Comment (teohhanhui) [inline api/Dockerfile]: Are you sure this works? We need to set `ping.path` in the php-fpm config: https://www.php.net/manual/en/install.fpm.configuration.php#ping.path

> If this value is not set, no URI will be recognized as a ping page.

File: api/Dockerfile (hunk 2 above)
Comment (teohhanhui) [reply api/Dockerfile]: I'd suggest 1 tab for `CMD`, and 2 tabs for each line of the command. WDYT?

Comment (teohhanhui) [reply api/Dockerfile]: Looking at the official healthcheck examples: https://github.com/docker-library/healthcheck

We should use a separate script file instead.

**Comment by teohhanhui** [comment]
DX impact: the container's log would be filled with lines like:

```
127.0.0.1 -  19/Nov/2019:17:03:27 +0000 "GET /ping" 200
```

**Comment by teohhanhui** [comment]
Thanks @patrick-mota! :tada:

**Pull request merged**